cfgov/jinja2/v1/_includes/molecules/text-introduction.html

```diff
@@ -29,10 +31,19 @@
                         at the bottom of the molecule.
 
    ========================================================================== #}
-
+{% if value.eyebrow and value.heading %}
+<header>
+{% endif %}
+{% if value.eyebrow %}
+    <div class="h5 eyebrow">{{ value.eyebrow }}</div>
```

Comment:
What would you think about removing the `h5` class and applying it via mixin to `.eyebrow`?

Reply:
That's a good question. Could there ever be a non-H5 eyebrow, or is that styling a fundamental characteristic of an eyebrow in our usage?

Reply:
That's also a good question :) Perhaps you could check with your designers for their input?

Reply:
I have asked for their feedback on this & will update when I have more info :)

Reply:
@Scotchester It sounds like eyebrows will always be H5, so I went with the mixin approach in https://github.com/cfpb/cfgov-refresh/pull/4004/commits/844e3834a04973256ef90343ea2db8b3111fada8 -- thanks for the suggestion!